es-data.yaml

```diff
@@ -56,7 +58,11 @@ spec:
         volumeMounts:
         - name: storage
           mountPath: /data
-      volumes:
-          - emptyDir:
-              medium: ""
-            name: "storage"
+  volumeClaimTemplates:
+  - metadata:
+      name: storage
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      resources:
+        requests:
+          storage: 12Gi
```

Comment:
Where does this volume comes from? Can you provide context and documentation on how to leverage on this, please?